html/login.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"/>
+  <title>login</title>
+  <link rel="stylesheet" href="../css/loginSignUp.css">
+  <link rel="icon" href="../img/pandaLogoIcon.svg">
+</head>
+<body>
+  <main>
+    <a class="main-logo-contain" href="../index.html">
```

Comment:
💊 제안 
로고를 눌렀을 때는 대부분 루트로 가니 지금 구조에서는 "/"로 변경하시는게 더 직관적일 것 같아요~

```suggestion
    <a class="main-logo-contain" href="/">
```